subsystems/drivebase/SwerveDrive.java

```diff
@@ -1,11 +1,14 @@
 package frc.team670.mustanglib.subsystems.drivebase;
 
 import java.util.Map;
+
+import org.littletonrobotics.junction.Logger;
+
 import com.pathplanner.lib.PathPlannerTrajectory;
 import com.pathplanner.lib.auto.SwerveAutoBuilder;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMax.IdleMode;
-
+import frc.team670.mustanglib.subsystems.VisionSubsystemBase;
```

Comment:
why was this added? I don't see it being used in any of the new code below. Did I miss it? If not, you can probably configure your editor to clean out unnecessary imports